ui/token/TokenPageTitle.tsx

```diff
@@ -98,7 +98,7 @@ const TokenPageTitle = ({ tokenQuery, addressQuery, hash }: Props) => {
       { verifiedInfoQuery.data?.tokenAddress && (
         <Tooltip label={ `Information on this token has been verified by ${ config.chain.name }` }>
           <Box boxSize={ 6 }>
-            <IconSvg name="verified_token" color="green.500" boxSize={ 6 } cursor="pointer"/>
+            <IconSvg name="certified" color="green.500" boxSize={ 6 } cursor="pointer"/>
```

Comment:
Can we use the `ui/shared/ContractCertifiedLabel.tsx` component here?

Reply:
it has a different tooltip text. I would not join them, logically they have nothing in common.